frontend/src/scenes/integrations/gitlab/gitlabSetupModalLogic.ts

```diff
@@ -0,0 +1,54 @@
+import { connect, kea, path, props } from 'kea'
+import { forms } from 'kea-forms'
+
+import api from 'lib/api'
+import { integrationsLogic } from 'lib/integrations/integrationsLogic'
+import { lemonToast } from 'lib/lemon-ui/LemonToast'
+
+import type { gitlabSetupModalLogicType } from './gitlabSetupModalLogicType'
+
+export interface GitLabSetupModalLogicProps {
+    isOpen: boolean
+    onComplete: (integrationId?: number) => void
+}
+
+export const gitlabSetupModalLogic = kea<gitlabSetupModalLogicType>([
+    path(['integrations', 'gitlab', 'gitlabSetupModalLogic']),
+    props({} as GitLabSetupModalLogicProps),
+    connect(() => ({
+        values: [integrationsLogic, ['integrations', 'integrationsLoading']],
```

Comment:
Is it used anywhere?

Reply:
Good spot, copy paste error from some other logics